packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -1662,6 +1662,122 @@ describe("IModelTransformerHub", () => {
     sinon.restore();
   });
 
+  it("should be able to handle a transformation which deletes a relationship and then elements of that relationship", async () => {
+    const masterIModelName = "Master";
+    const masterSeedFileName = path.join(outputDir, `${masterIModelName}.bim`);
+    if (IModelJsFs.existsSync(masterSeedFileName))
+      IModelJsFs.removeSync(masterSeedFileName);
+    // userlabel / codevalue: set json property named updateState on the physicalobject to value (simplified state updates)
```

Comment:
nit: I would prefer you add an explicit type (IModelTimelineElementDelta iirc from the TimelineTestUtils) that documents this fact, rather than comment it here

Reply:
done